python/hail/expr/expression.py

```diff
@@ -3325,32 +3465,32 @@ def one_hot_alleles(self, v):
     def one_hot_genotype(self, v):
         """Returns the triangle number of the genotype one-hot encoded into an integer array.
 
+        Valid only for diploid calls (``ploidy == 2``).
```

Comment:
I'm also OK with deleting this.  Otherwise, I think "valid" is a little vague here, say we throw an error if it is not diploid.